### DISCUSSION TITLE
config.json not loaded

### DISCUSSION THREAD

**Discussion by patofet** [Pryaxis/TShock#1806] [Q&A]
Hi, I'm trying to make a server with friends and I want to put a password in order to access the server, first of all I have the server in a raspberry pi 4, I simply installed mono and download "TShock_4.4.0_226_Pre1_Terraria1.4.0.2.zip" I put all the files in "/home/pi/TerrariaS" and start the server `mono ./TerrariaServer.exe  -port 7777 -world "/home/pi/.local/share/Terraria/Worlds/World.wld" -config config.json`
All work as expected(I can connect to it) but the server doesn't ask for passwort, I have to set it manually every time that I start the server.

I noticed that the first time that I start the server it doesn't create the "config.json" and I created it manually:
```
{
  "ServerPassword": "12345"
}
```
Someone knows how to start the server with a password automaticly?

**Comment by hakusaro** [comment]
Can you update to pre4? You're 3 versions behind.
